scss/lib/_lock.scss

```diff
@@ -27,5 +28,5 @@
     $min-breakpoint: convert(px, $min-breakpoint, $root: true);
     $max-breakpoint: convert(px, $max-breakpoint, $root: true);
 
-    @return calc(#{$min-size} + #{strip-units($max-size - $min-size)} * ((100vw - #{$min-breakpoint}) / #{strip-units($max-breakpoint - $min-breakpoint)}));
+    @return calc(#{$min-size} + #{strip-units($max-size - $min-size)} * (math.div((100vw - #{$min-breakpoint}), #{strip-units($max-breakpoint - $min-breakpoint)})));
```

Comment:
we wouldn't use math.div here because we don't want sass to process this - we want it to spit out a css `calc()` function